notebooks/122-quantizing-model-with-accuracy-control/122-speech-recognition-quantization-wav2vec2.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>header is misleading, because you compare accuracy, not performance.</p><p>And also probably, it will be useful if you compare FP16 model too, to see that there is no degradation after model conversion</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>